utils/channel_utils.tsx

```diff
@@ -98,3 +109,23 @@ export function joinPrivateChannelPrompt(team, channel, handleOnCancel = true) {
         return result;
     };
 }
+
+export function makeNewEmptyChannel(displayName: string, teamId: string): Channel {
+    return {
+        team_id: teamId,
+        name: displayName.split(' ').join('-').toLowerCase(),
```

Comment:
Do we have a tool/function that does some normalization? Just wondering if there's a way that some non-desired characters end up here (eg cyrilic or greek letters)